updated-gmailbigquery (#978)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/gmail-bigquery.md

## Setup and Configuration

The integration requires the following information to connect to Gmail BigQuery:
* `name`
* `projectId`
* `datasetId`
* `privateKey`
* `clientEmail`
* `tokenURI`
* `dataLocation`

:::info

> * User can find `projectId`, `privateKey`, `clientEmail` and `tokenURI` from the Google Cloud Console

Reviewer: Our style is to use "you", not "user"
```suggestion
* You can find `projectId`, `privateKey`, `clientEmail` and `tokenURI` from the Google Cloud Console
```

* BigQuery DataSetId  = `datasetId`
* BigQuery DataSet Location  = `dataLocation`

> * Default value of startTime is "24 Hours ago", user can change it later

Reviewer: ```suggestion
* You can change the `startTime`, if desired (default value is "24 Hours ago")
```

Author: Thanks for the suggestions. As these are not complete clauses so in order to maintain consistency. We can decide what goes with the doc. I prefer to keep it as is.

Reviewer: Regardless, it's still our style to use "you", not "user". Please update those.

cc @stacykor
